File: C/Program09.c
/////////////////////////////////////////////////////////////////////////
//                                                                     //
//  File name :     Program9.c                                         //
//  Description :   Used to find summation of natural numbers upto N   //
//  Author :        Pallavi Omprakash Malewar                          //
//  Date :          28/04/2025                                         //
//                                                                     //
/////////////////////////////////////////////////////////////////////////

#include<stdio.h>

int Summation(int iNumber)
{
    int iCount = 0;
    int iSum = 0;

    for(iCount = 1 ; iCount <= iNumber ; iCount++)
    {
        iSum = iSum + iCount;
    }

    return iSum;

}


int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number : ");
    scanf("%d" , &iValue);

    iRet = Summation(iValue);

    printf("Summation is : %d\n" , iRet);

    return 0;
}

File: JAVA/Program09.java
///////////////////////////////////////////////////////////////////////////////
//                                                                           //
//  File name :     Program09.java                                           //
//  Description :   Used to Display Sum of Non-Factors of Given Number       //
//  Author :        Pallavi Omprakash Malewar                                //
//  Date :          24/08/2025                                               //
//                                                                           //
///////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Number
{
    public int sumOfNonFactors(int iNo)
    {
        int iSum = 0;

        for(int iCount = 1 ; iCount <= iNo ; iCount++)
        {
            if((iNo % iCount) != 0)
            {
                iSum += iCount;
            }
        }
        return iSum;
    }
}

class Program09
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0, iRet = 0;

        System.out.print("Enter Number : ");
        iValue = sobj.nextInt();

        Number nobj = new Number();

        iRet = nobj.sumOfNonFactors(iValue);

        System.out.println("Sum of Non Factors of "  + iValue + " is " + iRet);

        sobj.close();
    }
}